pkg/models/suite_test.go

```diff
@@ -0,0 +1,127 @@
+package models
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/content-services/content-sources-backend/pkg/config"
+	"github.com/lib/pq"
+	"github.com/openlyinc/pointy"
+	"github.com/stretchr/testify/suite"
+	"gorm.io/driver/postgres"
+	"gorm.io/gorm"
+)
+
+type ModelsSuite struct {
+	suite.Suite
+	db *gorm.DB
+	tx *gorm.DB
+}
+
+func getDSNWithOptions(user string, password string, dbname string, host string, port int) string {
+	return fmt.Sprintf(
+		"user=%s password=%s dbname=%s host=%s port=%d sslmode=disable",
+		user,
+		password,
+		dbname,
+		host,
+		port,
+	)
+}
+
+func getDSNWithConfig(c *config.Configuration) string {
+	if c == nil {
+		return ""
+	}
+	return getDSNWithOptions(
+		c.Database.User,
+		c.Database.Password,
+		c.Database.Name,
+		c.Database.Host,
+		c.Database.Port,
+	)
+}
+
+func getDSNDefault() string {
+	config := config.Get()
+	return getDSNWithConfig(config)
+}
+
+func getDbConnection() *gorm.DB {
+	dsn := getDSNDefault()
+	db, err := gorm.Open(postgres.Open(dsn), &gorm.Config{})
+	if err != nil {
+		return nil
+	}
+	return db
+}
+
+const orgIdTest = "acme"
+const accountIdTest = "817342"
+
+var repoConfigTest1 = RepositoryConfiguration{
+	Base: Base{
+		// UUID generated by `uuid -v 4`
+		UUID:      "67eb30d9-9264-4726-9d90-8959e0945a55",
+		CreatedAt: time.Now(),
+		UpdatedAt: time.Now(),
+	},
```

Comment:
Base will be created when Create is called, so I don't think specifying it here is necessary. In all the places I've found that you've used these demo models, you're calling create, and these values are being overwritten anyway.

Reply:
Thanks! Updated!